src/main/java/com/aws/greengrass/authorization/WildcardTrie.java

```diff
@@ -149,6 +161,23 @@ private WildcardTrie add(String subject, boolean isTerminal) {
         return current;
     }
 
+    /**
+     * The method tries to parse the given string using escape sequence ${c} (where c is a character to be escaped)
+     * and returns the character c if the pattern is matched. In any other scenario it returns null character ('\0')
+     *
+     * @param str string provided to get
+     */
+    public static char getActualChar(String str) {
```

Comment:
why public? can this be package private?

Reply:
done.